src/KafkaFlow/Compressor/IMessageDecompressor.cs

```diff
@@ -0,0 +1,15 @@
+namespace KafkaFlow.Compressor
```

Comment:
Honest doubt: Shouldn't it be `KafkaFlow.Decompressor`?

Reply:
Same answer as the one in the Serializer